pug/invoice.pug

```diff
@@ -115,7 +115,7 @@ html(lang="en" ng-app="3drepo")
 
 				#billingInvoice.topBlocks
 					div
-						img#billingInvoiceLogo(src=baseURL+"/images/logo.png")
+						img#billingInvoiceLogo(src=baseURL+"/images/3drepo-logo-blue.png")
```

Comment:
I don't think the blue logo is in the repo. If that's the one we're using you need to add it in.
@jozefdobos in the past we were using the black and white one. Do you want the blue one? or you want to use the black solid logo?